core/healthcheck.go

```diff
@@ -34,7 +34,9 @@ func (d *portHealthChecker) Check(ctx context.Context) (err error) {
 
 	args := []string{"check", d.host}
 	for _, port := range d.ports {
-		args = append(args, fmt.Sprintf("%d/%s", port.Port, port.Protocol.Network()))
+		if !port.SkipHealthCheck {
+			args = append(args, fmt.Sprintf("%d/%s", port.Port, port.Protocol.Network()))
+		}
```

Comment:
Seems like we'll need to handle the case where all ports are `SkipHealthCheck`. If that's the case we should just skip the whole process. I'm not sure if it does, but I could imagine the `check` command failing if it isn't given any ports to check. (This probably warrants an integration test of its own.)